app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -208,22 +209,29 @@ class MainActivity : ComponentActivity() {
                 Column() {
                     Row(modifier = Modifier
                         .fillMaxWidth()
-                        .background(Color(0xFFE0E0E0)), horizontalArrangement = Arrangement.SpaceBetween) {
+                        .background(Color(0xFFE0E0E0)),
+                        horizontalArrangement = Arrangement.SpaceBetween) {
                         //Close Button
-                        Button( onClick = { openDialog = false}, colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent), elevation = ButtonDefaults.elevation(0.dp, 0.dp)) {
+                        Button( onClick = { openDialog = false},
```

Comment:
Broke up UI properties for added readability to display different elements implementations, as well as to wrap lines that were too long.
https://developer.android.com/kotlin/style-guide#where_to_break